2021-08-13 20:46:12,253 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:12,501 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:12,660 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:12,748 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:12,986 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:13,119 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:13,186 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:50,795 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,069 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,178 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,268 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,544 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,670 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:46:52,797 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:43,321 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:43,560 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:43,959 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:44,554 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:44,800 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:45,174 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:48:45,288 test_03_recharge.py-88:root:ERROR - 测试用例失败
2021-08-13 20:49:12,950 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:14,494 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:14,596 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:18,182 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:18,364 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:18,495 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:49:18,620 test_03_recharge.py-89:root:ERROR - 测试用例失败
2021-08-13 20:51:04,847 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,039 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,107 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,173 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,338 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,463 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:51:05,598 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:53:46,175 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:53:46,415 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:53:46,477 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:53:46,570 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:53:46,646 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:53:46,857 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:53:47,090 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:35,345 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:54:35,549 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:35,657 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:35,949 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:37,243 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:37,453 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:37,559 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:58,128 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:54:58,330 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:58,441 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:58,522 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:58,663 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:54:58,850 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:54:58,951 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:07,503 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:07,630 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:07,757 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:07,949 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:08,152 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:08,272 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:08,348 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:14,107 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:14,326 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:14,427 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:14,536 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:14,721 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:14,957 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:15,122 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:18,283 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:18,503 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:18,599 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:18,672 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:18,834 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-08-13 20:56:18,995 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-13 20:56:19,059 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-08-18 20:23:39,026 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-08-18 20:23:39,161 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-18 20:23:39,283 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-18 20:23:39,385 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-18 20:23:39,469 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-08-18 20:23:39,693 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-18 20:23:39,803 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-25 20:21:36,776 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-08-25 20:21:37,154 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-25 20:21:37,284 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-25 20:21:37,446 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-25 20:21:37,690 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-08-25 20:21:37,969 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-25 20:21:38,082 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-08-26 21:04:09,248 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:08:54,932 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:09:27,940 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:11:40,607 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:16:06,381 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:17:21,081 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:21:22,087 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:29:30,517 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:30:15,468 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:45:29,064 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-08-27 20:46:33,314 test_04_invest.py-43:root:ERROR - 测试用例失败
2021-08-31 19:59:02,814 test_04_invest.py-42:root:ERROR - 测试用例失败
2021-09-04 10:02:24,150 test_04_invest.py-63:root:ERROR - 测试用例失败
2021-09-04 10:48:29,307 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:49:48,110 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:56:53,389 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:17,715 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:18,338 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:19,097 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:21,882 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:22,674 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:23,049 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:23,438 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 10:58:23,847 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:47,291 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:47,713 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:48,440 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:50,836 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:51,139 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:51,559 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:52,034 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:52,421 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:10:52,896 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:31,702 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:32,332 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:33,219 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:35,029 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:35,963 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:36,355 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:36,775 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:37,272 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:37,711 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-04 11:29:38,438 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:45,921 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:46,393 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:47,271 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:49,369 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:50,488 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:50,906 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:51,332 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:52,047 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:52,574 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:09:53,139 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:41,724 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:42,118 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:42,784 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:44,673 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:45,930 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:47,872 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:48,446 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:49,105 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:49,525 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:12:50,361 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:04,046 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:04,413 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:05,213 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:07,147 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:07,764 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:08,264 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:08,694 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:09,409 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:09,944 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:10,640 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:13:11,184 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:39,165 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:39,610 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:40,388 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:41,877 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:42,526 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:43,124 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:43,786 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:44,285 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:44,676 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:45,142 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:45,828 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 20:17:46,295 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:41,036 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:41,494 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:42,206 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:44,280 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:44,814 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:45,612 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:46,126 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:46,519 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:47,014 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:47,526 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:47,886 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:03:48,351 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:12,746 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:12,868 test_01_login.py-55:root:ERROR - 测试用例失败： 0 != 1
2021-09-05 21:04:12,937 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:12,963 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,025 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,049 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,071 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,254 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,278 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,340 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,428 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,454 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,476 test_01_login.py-55:root:ERROR - 测试用例失败： 2 != 1
2021-09-05 21:04:13,747 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:13,915 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:14,151 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:14,545 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:14,860 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:15,110 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:15,357 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-05 21:04:16,062 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-09-05 21:04:16,400 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:16,520 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:16,765 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:17,086 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:17,489 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:17,885 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-05 21:04:19,106 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:19,553 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:20,566 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:21,883 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:22,563 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:23,148 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:23,705 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:24,082 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:24,763 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:25,267 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:25,688 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:04:26,065 test_04_invest.py-64:root:ERROR - 测试用例失败
2021-09-05 21:46:37,707 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:38,162 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:38,939 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:40,347 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:40,730 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:41,259 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:41,704 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:42,235 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:42,743 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:43,361 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:43,836 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-05 21:46:44,290 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-06 21:44:39,362 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-06 21:44:39,728 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:44:39,918 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:44:39,943 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:44:39,966 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:44:53,464 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-06 21:44:53,806 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:44:53,931 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:44:54,038 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:44:54,128 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:44:59,972 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-06 21:45:00,306 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:45:00,440 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:45:00,469 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:45:00,498 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:45:04,886 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-06 21:45:05,415 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:45:05,634 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-06 21:45:05,787 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-06 21:45:05,816 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-07 21:05:05,431 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-07 21:05:05,834 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-07 21:05:05,973 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-07 21:05:05,996 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-07 21:05:06,016 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:16:58,904 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-08 20:16:59,237 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:16:59,360 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:16:59,385 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:16:59,514 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:17:01,172 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-09-08 20:17:01,540 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:17:01,647 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:17:01,881 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:17:02,026 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-09-08 20:17:02,362 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:17:02,548 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:17:03,841 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:04,308 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:04,952 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:06,237 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:06,720 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:07,485 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:07,872 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:08,229 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:08,648 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:09,144 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:09,866 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:17:10,352 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:12,697 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-08 20:18:13,169 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:18:13,288 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:18:13,334 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:18:13,382 test_01_login.py-63:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:18:13,517 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:13,732 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:13,963 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:14,148 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:14,373 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:14,537 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:14,688 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:18:15,226 test_03_recharge.py-88:root:INFO - 测试用例成功
2021-09-08 20:18:15,672 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:15,798 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:16,058 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:16,309 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:16,787 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:16,928 test_03_recharge.py-91:root:ERROR - 测试用例失败
2021-09-08 20:18:18,205 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:19,091 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:19,883 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:21,547 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:22,094 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:22,997 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:23,405 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:23,736 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:24,199 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:24,697 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:25,176 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:18:25,777 test_04_invest.py-65:root:ERROR - 测试用例失败
2021-09-08 20:20:44,388 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-08 20:20:44,699 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:20:44,847 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:20:44,872 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:20:44,895 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:24:52,463 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-08 20:24:52,819 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:24:52,996 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-08 20:24:53,112 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:24:53,195 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-08 20:24:53,357 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:53,582 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:53,743 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:53,998 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:54,465 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:54,607 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:54,804 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:55,195 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-09-08 20:24:55,546 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-09-08 20:24:55,843 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-09-08 20:24:56,020 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-09-08 20:24:56,308 test_03_recharge.py-87:root:INFO - 测试用例成功
2021-09-08 20:24:56,675 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-09-08 20:24:57,012 test_03_recharge.py-90:root:ERROR - 测试用例失败
2021-09-08 20:24:59,471 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:24:59,911 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:00,696 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:02,628 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:03,186 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:03,957 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:04,390 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:04,849 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:05,446 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:05,966 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:06,516 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-08 20:25:07,096 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:11,673 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-25 20:49:12,093 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-25 20:49:12,339 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-25 20:49:12,459 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-25 20:49:12,686 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-25 20:49:12,968 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:13,253 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:13,675 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:24,023 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:24,343 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:49:24,613 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:49,747 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:50,514 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:50,911 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:51,262 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:51,606 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:51,996 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 20:58:52,386 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:15,923 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:16,278 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:16,597 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:16,947 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:17,195 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:17,445 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:00:17,867 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:28,243 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:28,629 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:29,547 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:29,870 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:31,539 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:32,183 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:32,520 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:01:32,935 test_02_register.py-66:root:ERROR - 测试用例失败
2021-09-25 21:06:16,008 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:16,582 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:18,708 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:19,121 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:21,932 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:22,251 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:22,618 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:06:22,952 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:21,275 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:21,847 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:22,947 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:23,294 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:25,080 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:25,409 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:25,681 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:07:25,976 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:08:00,122 test_03_recharge.py-89:root:INFO - 测试用例成功
2021-09-25 21:08:00,454 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:08:01,815 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:08:02,023 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:08:02,332 test_03_recharge.py-89:root:INFO - 测试用例成功
2021-09-25 21:08:02,747 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:08:03,149 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:08:19,967 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:20,593 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:21,396 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:22,970 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:23,547 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:24,168 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:24,948 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:26,413 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:27,061 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:27,564 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:28,056 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:08:28,648 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:10,143 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-25 21:09:10,719 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-25 21:09:10,971 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-25 21:09:11,090 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-25 21:09:11,225 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-25 21:09:11,962 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:12,404 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:13,283 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:13,624 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:15,467 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:15,847 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:16,165 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:16,608 test_02_register.py-68:root:ERROR - 测试用例失败
2021-09-25 21:09:17,117 test_03_recharge.py-89:root:INFO - 测试用例成功
2021-09-25 21:09:17,538 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:17,791 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:18,213 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:18,506 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:18,818 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:19,029 test_03_recharge.py-92:root:ERROR - 测试用例失败
2021-09-25 21:09:20,435 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:20,914 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:21,822 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:23,312 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:23,819 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:24,570 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:25,058 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:25,686 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:26,209 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:26,860 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:27,407 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:09:28,012 test_04_invest.py-66:root:ERROR - 测试用例失败
2021-09-25 21:21:37,179 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 2 != 0
2021-09-25 21:21:37,790 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-25 21:21:38,038 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1001 != 0
2021-09-25 21:21:38,173 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
2021-09-25 21:21:38,338 test_01_login.py-64:root:ERROR - 测试用例失败，报错信息： 1 != 0
